packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -490,6 +491,19 @@ class TextStyle extends Diagnosticable {
   /// The style in which to paint the text decorations (e.g., dashed).
   final TextDecorationStyle decorationStyle;
 
+  /// The thickness of the decoration stroke as a muliplier of the thickness
+  /// defined by the font.
+  ///
+  /// The font provides a base stroke width for decorations which scales off
```

Comment:
This doc would probably be slightly more accessible if it gave one or two concrete examples of what a decoration stroke may be that's affected by this property.